build/gulpfile.coffee

```diff
@@ -246,7 +246,7 @@ gulp.task 'karma', (done) ->
 			configFile: KARMA_CONFIG_FILE
 			singleRun: false
 			autoWatch: true
-		, done
+		, done()
```

Comment:
this will resolve the task immediately. i came up with a slightly better solution:
https://github.com/fullcalendar/fullcalendar-scheduler/commit/380c9c3735d814705e2cb3205a5fb10bfa051647
(pass a PITA to figure out)